docs/static-run.md

```diff
@@ -20,6 +21,7 @@ To run all standard tests, from the controller:
 
 ```console
 export TESTHOST="${PREFIX}suma3pg.tf.local"
+export PROXY="None"
```

Comment:
https://github.com/moio/sumaform/pull/202/commits/f9c4d20d92610489acafd8af243b01ee6aa86181#diff-0bf3346ebbff8e21e0e418df485ea0c6R20

Reply:
it's what salt gives when the vm does not exist